### ISSUE TITLE
shadowcoin in coinomi wallet not work

### ISSUE THREAD

**Issue by cryptomate** [Coinomi/coinomi-android#96]
l just get answer from faucet support that shadow coins server ok and it is wallet issue.
In my wallet shadow coins show all the time
"no connection" 
ver 1.5.22 
also under settling I can't see option  " to import a private key / sweep a paper wallet."
But...there NO SUCH OPTION EXIST .

**Comment by cryptomate** [comment]
I'm just get answer from support, that I need update my wallet version, so I did it.
l can see coinomi tram work hard, in new version many new futures, and this is grate!
I' really appreciate coinomi team for quick response!
unfortunately, in new version,(1.6.2) shadow coins also not work, it is all the time " no connection"
So I'm no idea why and what to do...

**Comment by cryptomate** [comment]
sorry,for mistaping, bad eyes,shaking hands,disability...

**Comment by cryptomate** [comment]
well, it is a week now, shadowCash is not work...
I checked few times,  all other currencies works fine.
transfers was sent l chech all.
And no any responce from anybody...
Is only my wallet sadows not work or someone else?

**Comment by mick93** [comment]
I have same problem

**Comment by Cr3atur3** [comment]
Yeah same here no connection but it never asked 4 public key and nit give one for me problemishes I'd send to wallet address and with internal exchange
